src/main/kotlin/com/cognifide/gradle/aem/common/instance/Ims.kt

```diff
@@ -30,6 +30,10 @@ class Ims(private val aem: AemExtension) {
      */
     private val expirationTimeNormalized get() = System.currentTimeMillis() / 1000 + expirationTime.get()
 
+    val lastExpirationTimeUsed = aem.obj.long {
```

Comment:
I don't get it how that in-memory caching may improve something.
remember that Ims object is instantiated separately for each Gradle project/ module so rather some OS/file caching may be better here

Reply:
I set this because I want to export the expiration time of the generated token to save that also in the instance object, and anyway in-memory or file/OS caching, both cases work but I'm not where you mean by that, could you elaborate? or is point to another place in gap where you do that?